src/validate.ts

```diff
@@ -178,6 +178,12 @@ export function response(compiledPath: CompiledPath | undefined,
   if (response === undefined) {
     response = operation.responses['default'];
   }
+  if (response === undefined) {
```

Comment:
Actually on second reading there's really no specified behavior for this situation.  I'll merge this PR but may move the error check to the `validate` function.  Thanks!